src/mbgl/tile/tile_cache.cpp

```diff
@@ -33,13 +33,22 @@ void TileCache::add(const OverscaledTileID& key, std::unique_ptr<Tile> tile) {
 
     // purge oldest key/tile if necessary
     if (orderedKeys.size() > size) {
-        get(orderedKeys.front());
+        pop(orderedKeys.front());
     }
 
     assert(orderedKeys.size() <= size);
 }
 
-std::unique_ptr<Tile> TileCache::get(const OverscaledTileID& key) {
+Tile* TileCache::get(const OverscaledTileID& key) {
+    auto it = tiles.find(key);
+    if (it != tiles.end()) {
+        return it->second.get();
+    } else {
+        return nullptr;
+    }
+}
+
+std::unique_ptr<Tile> TileCache::pop(const OverscaledTileID& key) {
```

Comment:
Did you check other uses of `::pop` to make sure we're not accidentally changing existing behavior?

Reply:
Yes, I replaced all preexisting calls to `TileCache::get`(old name for this `pop` function) with `TileCache::pop` and `TileCache::get` is only used to backfill raster-dem tiles